### ISSUE TITLE
Accessibility] [Keyboard Navigation -Playground]: Keyboard focus struck in Code pane in playground page. [Sev2][WCAG2.1.2]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#38]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now, navigate to the ‘Playground’ link and activate it.
4.	Now, navigate to code pane in playground page and check whether the keyboard focus struck in the pane or not.

**Actual:**
Keyboard focus struck in Code pane in playground page. User is not able to navigate out of the code pane.

**Expected:**
Keyboard focus should not struck in Code pane in playground page. User should be able to navigate out of the code pane. There is should be a way to navigate out like by using ‘ESC’ key.
Keyboard focus should not get stuck on a control. This issue is usually caused by JavaScript interference which prevents keyboard tab navigation.

**User Impact:**
When focus is trapped on a control, if interferes with the user’s ability to interact with the whole page/application. This can include content that requires multiple tabs/swipes to move focus, or situations where a user can’t tab/swipe past in either direction.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/no-keyboard-trap

https://user-images.githubusercontent.com/88436727/150645062-2924cbe2-298e-408e-af93-41a0a9e9356a.mp4
